frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -53,7 +53,7 @@ function FileManagerMenu:initGesListener()
     self.ges_events = {
         TapShowMenu = {
             GestureRange:new{
-                ges = "tap",
+                ges = "swipe",
```

Comment:
We need to migrate this to the touch zone framework and override `rolling_swipe`'s zone since they are overlapping.